framework/source/class/qx/theme/manager/Appearance.js

```diff
@@ -286,6 +286,38 @@ qx.Class.define("qx.theme.manager.Appearance",
         // Create new map
         result = {};
 
+        // handler for group properties
```

Comment:
This `groupHandler` function seems to be the same implementation as the one in Decoration.js#L228; please can you refactor this to a single function, possibly one that includes the `if (qx.core.Property.$$groups[baseIncludeKey]) {` bit as well?

Reply:
Done. Thank you!